docs/content/guides/2.alokai-essentials/1.alokai-next-js/4.install-sdk.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Install Alokai SDK
+layout: default
+navigation:
+  icon: tabler:number-3-small
+---
+
+# Install and Configure Alokai SDK
+
+In the last sections, we have configured Alokai Next.js project and installed the Alokai Middleware. In this section, we will install and configure the Alokai SDK.
+
+The Alokai SDK and Alokai Middleware together called Alokai Connect. Both SDK and Middleware are used in tandem to connect the Alokai frontend with the backend. SDK is responsible for establishing the connection with the Middleware, while Middleware is responsible to orchestrate the communication with the backend.
+
+In this guide, we will install the Alokai SDK and configure it to work with SAP Commerce Cloud and Alokai Middleware.
+
+## Install Alokai SDK
+
+This time we will solely focus on `storefront` application. Navigate to a newly generated `storefront` directory and install the Next.js SDK package by running the following command:
+
+```bash
+cd apps/storefront
+npm install --save-dev @vue-storefront/next
+```
+
+It is a dedicated Next.js package for Alokai SDK. It simplifies the use of the SDK across the React Server Components and Client Components.
+
+Next, you need to install a special integration package for SAP Commerce Cloud. This package is an extension of the core SDK and provides additional functionality for SAP Commerce Cloud.
+
+SAP Commerce Cloud integration is part of our Enterprise offering. To access this package, you will need to have Alokai license to access to our private npm registry.
+
+:::info
+If you don't have access to our private npm registry, please [contact our sales team](https://vuestorefront.io/contact/sales).
+:::
+
+We already added the private npm registry to the `.npmrc` file in the previous section. Now you can install the `@vsf-enterprise/sapcc-sdk` package by running the following command from `apps/storefront` directory:
+
+```bash
+npm install @vsf-enterprise/sapcc-sdk
+```
+
+And that's it! You have successfully installed the Alokai SDK and SAP Commerce Cloud integration. 
+
+## Configure Alokai SDK
+
+Now that you have successfully installed the Alokai SDK and SAP Commerce Cloud integration, you need to configure the SDK. 
+
+Create a new directory in the `apps/storefront` directory called `sdk`. Inside the `sdk` directory, create a new file called `sdk.config.ts` and add the following code:
+
+```typescript
+import { sapccModule } from '@vsf-enterprise/sapcc-sdk';
```

Comment:
This code snipped could be updated as right now we would like to use `middlewareModule` for that :)